package.json

```diff
@@ -24,6 +24,9 @@
   "pnpm": {
     "overrides": {
       "@solana/web3.js@1.77.4>rpc-websockets": "7.11.0"
+    },
+    "patchedDependencies": {
+      "eccrypto@1.1.6": "patches/eccrypto@1.1.6.patch"
```

Comment:
I removed this patch in #2489 and the proposals app works in prod without it, but I didn't test local dev mode, and there  without this patch we get a syntax error in eccrypto, so I need to resurrect it.